src/Graph.cpp

```diff
@@ -2107,13 +2110,15 @@ void Graph::rewireRandomEdges(double rewiredEdgesProportion) {
 }
 
 string Graph::autogenFilesFolder() {
-    return "networks/"+name+"/autogenerated/";
+    return "autogenerated/";
```

Comment:
Aha. Here it is, defined. So you should call autogenFilesFolder() whenever you need it, rather than hard-coding the string "autogenerated" everywhere.